### PR TITLE
Label change

### PR DESCRIPTION
**Pull request by viktorfrom**

### CODE REVIEW

File: simple/datasets/email/training_data.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # training data: texts, heads and dependency labels
 # for no relation, we simply chose an arbitrary dependency label, e.g. '-'
 
-# Labels: [ROOT, NOUN, TO, WHEN, BODY]
+# Labels: [ROOT, NOUN, TO, START, BODY]
```
Comment (markhakansson) [inline simple/datasets/email/training_data.py]: Don't change other datasets

File: simple/datasets/reminder/training_data.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # training data: texts, heads and dependency labels
 # for no relation, we simply chose an arbitrary dependency label, e.g. '-'
 #
-# Labels: [ROOT, NOUN, TO, WHEN, BODY]
+# Labels: [ROOT, NOUN, TO, START, BODY]
```
Comment (markhakansson) [inline simple/datasets/reminder/training_data.py:4]: Same here don't change other datasets

File: deep/seq2seq.py [hunk 3]
```diff
@@ -113,15 +113,15 @@ def correct_vector_length(vectors, target_length, pad_vector):
     dataset_outputs = ["email <TO> alemen-6@student.ltu.se <BODY> Hello world!",
                        "email <TO> my friend <BODY> I'm sick I cannot come to school.",
                        "email <TO> my colleagues <BODY> I am about 15 minutes late for work.",
-                       "remind <TO> me <WHEN> 8am tomorrow <BODY> Take out the trash.",
-                       "remind <TO> me <WHEN> next week, monday, 5pm <BODY> Dentist appointment.",
-                       "remind <TO> my team <WHEN> friday <BODY> Meeting."]
+                       "remind <TO> me <START> 8am tomorrow <BODY> Take out the trash.",
```
Comment (markhakansson) [inline deep/seq2seq.py]: Don't change the model Alex is maintaining

**Review by markhakansson — CHANGES_REQUESTED** [review]

File: deep/seq2seq.py [hunk 4]
```diff
@@ -154,7 +154,7 @@ def correct_vector_length(vectors, target_length, pad_vector):
     # Training loop
     filename = "seq2seq_checkpoint.pt" # used for saving/loading trained models
     if True:
-        print("Training started")
+        print("Training WHENed")
```
Comment (markhakansson) [inline deep/seq2seq.py:157]: WTF

**Review by markhakansson — CHANGES_REQUESTED** [review]

File: simple/datasets/reminder/training_data.py [hunk 5]
```diff
@@ -135,7 +135,7 @@
         },
     ),
     (
-        "warn me when the time is 20.15",
+        "warn me WHEN the time is 20.15",
```
Comment (Blinningjr) [inline simple/datasets/reminder/training_data.py:138]: CAPS

**Review by Blinningjr — APPROVED** [review]

**Review by markhakansson — APPROVED** [review]

**Pull request merged**